pull-request.sh

```diff
@@ -105,12 +106,16 @@ create_pull_request() {
                 DATA="{\"assignees\":[${ASSIGNEES}]}"
                 echo "${DATA}"
                 ASSIGNEES_URL="${ISSUE_URL}/${NUMBER}/assignees"
-                curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X POST --data "${DATA}" ${ASSIGNEES_URL}
-                RETVAL=$?
+                RETVAL=0
+                if ! RESPONSE="$(safe_curl -X POST --data "${DATA}" ${ASSIGNEES_URL})"; then
+                    RETVAL=$?
+                fi
                 printf "Add assignees return code: ${RETVAL}\n"
                 echo ::set-env name=ASSIGNEES_RETURN_CODE::${RETVAL}
                 echo ::set-output name=assignees_return_code::${RETVAL}
-
+                if [[ $RETVAL != 0 ]]; then
+                    curl_die "Error adding assignees: $RESPONSE"
```

Comment:
For all variables, please format like `"${VARIABLE}"` with curly brackets for consistency.